### DISCUSSION TITLE
[RFE][API] Support sort and direction parameters on List issue comments endpoint

### DISCUSSION THREAD

**Discussion by benipeled** [community/community#76603] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

The `sort` and `direction` parameters are supported on [List issue comments for a repository](https://docs.github.com/en/rest/issues/comments?apiVersion=2022-11-28#list-issue-comments-for-a-repository) but missing from [List issue comments](https://docs.github.com/en/rest/issues/comments?apiVersion=2022-11-28#list-issue-comments),

In my case, I'm looking for the last x comments of a given issue but it's not straightforward in case of >100 comments,
It can be easily done with `sort=updated&direction=asc` (which isn't supported at the moment)

**Comment by isarns** [comment]
+1
